pyodata/v2/model.py

```diff
@@ -802,6 +805,9 @@ def list_associations(self):
         def list_association_sets(self):
             return list(self.association_sets.values())
 
+        def list__collections_entity_types(self):
```

Comment:
That double _ is not necessary, use just 1.

Reply:
typo... yeah